content/08-cardano-sidechains/03-example-evm-sidechain/01-network-details.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: Network details
+metaTitle: Network details
+---
+
+The Example EVM Sidechain testnet enviorment consistists of 2 networks:
```

Comment:
The Example EVM sidechain testnet environment consists of 2 networks:

Reply:
also 'two' instead of '2'. Neil, is it convenient to leave comments in GH? Maybe just push editing changes directly? @stevanlohja what do you think?

Reply:
I prefer to leave comments, so the original author can see what was changed.

Reply:
@olgahryniuk Your suggestion, "two", is much better.